app/services/xeno_canto_service.rb

```diff
@@ -0,0 +1,19 @@
+class XenoCantoService
+  attr_reader :connection
+
+  def initialize
+    @connection = Faraday.new(url:
+      "http://www.xeno-canto.org/api/2")
+  end
+
+  def recordings(min_lat, min_long, max_lat, max_long)
+    parse(connection
+      .get("recordings?query=box:#{min_lat},#{min_long},#{max_lat},#{max_long}"))
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.